doc/man/8/rados.rst

```diff
@@ -26,6 +26,12 @@ Options
 
    Interact with the given pool. Required by most commands.
 
+.. option:: --pgid
+
+   As an alternative to --pool, --pgid also allows user to specify the
+   PG id to which the command will be directed. With this option, certain
+   commands like ls allows user to limit the scope of the command to the given PG.
```

Comment:
could you quote "--pool", "--pgid" and "ls" with backquotes?

Reply:
thanks Kefu. done.